### PR TITLE
DPDK CPU pinning

### PR DESCRIPTION
**Pull request by bnshr**
DPDK pod deployment is based on the cluster setup, existence of HugePages volume, network. Hence, some labels / annotations are hard-coded.

### CODE REVIEW

**Comment by bnshr** [comment]
@theemj Could you please test this test case in QE cluster where the cluster setup is ready?

File: tests/networking/tests/networking_dpdk_cpu_pinning_exec_probe.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+	corev1 "k8s.io/api/core/v1"
+)
+
+/*
+	The setup needs to be as mentioned here :
+	https://developers.redhat.com/articles/2021/08/27/using-virtual-functions-dpdk-red-hat-openshift
+*/
+
+var _ = Describe("Networking dpdk-cpu-pinning-exec-probe,", func() {
```
Comment (sebrandon1) [inline tests/networking/tests/networking_dpdk_cpu_pinning_exec_probe.go:20]: Has this worked in a parallel execution?  We can put a `Serial` here if its not ready quite yet and then remove the `Serial` later.

File: tests/networking/tests/networking_dpdk_cpu_pinning_exec_probe.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+	})
+
+	It("one dpdk pod with exec probe [negative]", func() {
+
+		By("Deploy dpdk pod namespace")
+		dpdkPod := tshelper.DefineDpdkPod(tsparams.DpdkPodName, randomNamespace)
+
+		By("Redefine liveness probe")
+
+		containerLivenessProbe := &corev1.Probe{
```
Comment (theemj) [inline tests/networking/tests/networking_dpdk_cpu_pinning_exec_probe.go:72]: We do have a [RedefineWithLivenessProbe function for pods ](https://github.com/test-network-function/cnfcert-tests-verification/blob/4d6bc815c7db3767925a90149e79b9431849547d/tests/utils/pod/pod.go#L52)that could be used here, though it looks like we'd have to add a few params to the function to include all the values being set here. Either way works, might just keep things a little cleaner.

File: tests/networking/tests/networking_dpdk_cpu_pinning_exec_probe.go (hunk 1 above)
Comment (bnshr) [reply tests/networking/tests/networking_dpdk_cpu_pinning_exec_probe.go:20]: Initially I used with Serial when we had one pod deployment in default namespace. But now we moved to pod deployment in random namespace, hence parallel execution would work.

File: tests/networking/tests/networking_dpdk_cpu_pinning_exec_probe.go (hunk 2 above)
Comment (bnshr) [reply tests/networking/tests/networking_dpdk_cpu_pinning_exec_probe.go:72]: Yep. I would modify the function with commands parameter.

**Review by theemj — APPROVED** [review]

**Pull request merged**